ForumApi/Startup.cs

```diff
@@ -39,13 +39,14 @@ public void ConfigureServices(IServiceCollection services)
                 options.AddDefaultPolicy(
                                 builder =>
                                 {
-                                     builder.WithOrigins(
-                                         "http://20.94.137.143/", //Frontend
-                                         "http://20.189.29.112/", //Admintools
-                                         "http://20.45.2.119/" //User
-                                         )
-                                        .AllowAnyHeader()
-                                        .AllowAnyMethod();
+                                    builder.WithOrigins(
+                                        "http://20.94.137.143/", //Frontend
+                                        "http://20.189.29.112/", //Admintools
+                                        "http://20.45.2.119/", //User
+                                        "http://localhost:4200/"
```

Comment:
We need cors for testing locally since a lot of different frontend parts being worked on separately at the same time